schemata/organization.json

```diff
@@ -34,7 +34,7 @@
         "type": "object",
         "properties": {
           "id": {
-            "type": "integer",
+            "type": "bigint",
```

Comment:
@benjaminoakes it's not because this returns the organizations and their ID based on the user identity ID; it doesn't use the org ID in the path